### PR TITLE
bridge: Allow CockpitDBusJson to share introspection info

### PR DESCRIPTION
**Pull request by stefwalter**
I've split this out into its own commit in order to bisect
digressions that are happening with ordering.

### CODE REVIEW

File: src/bridge/cockpitdbuscache.c [hunk 1]
```diff
@@ -96,7 +96,7 @@ enum {
   PROP_CONNECTION = 1,
   PROP_NAME,
   PROP_LOGNAME,
-  PROP_NAME_OWNER
```
Comment (stefwalter) [inline src/bridge/cockpitdbuscache.c]: For the record, this line was not used anywhere ... that's why it's removed.

**Comment by stefwalter** [comment]
@mvollmer I need your help figuring out why a change like this makes the Networking bond and team tests fall over pretty reliably. The change is not that big, we're just moving the creation of a hashtable to be passed into ```CockpitDBusCache``` instead of being created directly in its ```_init()``` function

**Comment by stefwalter** [comment]
Marking this priority, since it's blocking about 10 other pull requests.

**Comment by mvollmer** [comment]
> @mvollmer I need your help figuring out why a change like this makes the Networking bond and team tests fall over pretty reliably.

For the record, this was because `cockpit-bridge` crashed with a SEGFAULT somewhere inside free(3) on a browser reload.  This is fixed by `bridge: Track DBus interfaces sent more correctly`.

The crash was not visible anywhere, I think  There was no coredump or any message in the journal.

**Pull request merged**